src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="col-sm-10 typeahead-dropdown pull-left">
+    <div *ngIf="!searchValue" (click)="open()" class="pointer details-dropdown" [ngClass]="getInitialValue().cssLabelClass?getInitialValue().cssLabelClass:''">{{ getInitialValue().value }}</div>
+    <div style="display: flex;">
+      <div *ngIf="searchValue" class="dropdown-search">
+        <div class="list-container">
+          <span class="fa fa-search dropdown-search-icon"> </span>
+          <input id="valueSearchInput" #valueSearch (keyup)="filterValue($event)" autocomplete="off" />
+          <ul class="item-ul dropdown-list" #valueList>
+            <li *ngIf="!values.length" class="textCenter"><i class="fa fa-spinner fa-pulse"></i></li>
```

Comment:
other areas of the UI use the PatternFly spinner (`<div class="spinner spinner-lg"></div>`). Size is changed by `spinner-lg / spinner-md / spinner-sm / spinner-xs`